.github/workflows/cmake.yml

```diff
@@ -0,0 +1,37 @@
+name: CMake
```

Comment:
Is this new workflow really needed? I believe there is already a CI for this...

Reply:
i didnt know i messed it up but it will looks like its messed up

Reply:
I would just remove the file